experimental/traffic-portal/src/app/core/statuses/statuses-table/statuses-table.component.html

```diff
@@ -0,0 +1,9 @@
+<mat-card class="table-page-content">
```

Comment:
This file is missing a license header. See other template HTML files for a comment block at the top that you can copy.